packages/markdown/components/Heading/index.jsx

```diff
@@ -1,26 +1,35 @@
 const React = require('react');
 const PropTypes = require('prop-types');
 
-function Heading({ tag, ...props }) {
+function Heading({ tag, opts, ...props }) {
   if (!props.children) return '';
 
+  const { splitReferenceDocs } = opts;
   const attrs = {
     className: `heading heading-${props.level} header-scroll`,
     align: props.align,
   };
-  return React.createElement(tag, attrs, [
+
+  const children = [
     <div key={`heading-anchor-${props.id}`} className="heading-anchor anchor waypoint" id={props.id} />,
     <div key={`heading-text-${props.id}`} className="heading-text">
       {props.children}
     </div>,
-    // eslint-disable-next-line jsx-a11y/anchor-has-content
-    <a key={`heading-anchor-icon-${props.id}`} className="heading-anchor-icon fa fa-anchor" href={`#${props.id}`} />,
-  ]);
+  ];
+
+  if (splitReferenceDocs) {
+    children.push(
+      // eslint-disable-next-line jsx-a11y/anchor-has-content
+      <a key={`heading-anchor-icon-${props.id}`} className="heading-anchor-icon fa fa-anchor" href={`#${props.id}`} />
+    );
+  }
```

Comment:
By this logic, wouldn't we end up hiding anchor icons on guides and custom pages too, since the default is false?

Reply:
That's a good point, you may be right!

Reply:
Fixed!